inference/core/models/roboflow.py

```diff
@@ -604,6 +605,9 @@ def __init__(
                             "trt_fp16_enable": True,
```

Comment:
Question: is there any value from making that parametrised by env?

Reply:
Enabling this flag uniformly made inference slower even for unconverted models in my tests